modules/docker/Makefile

```diff
@@ -1 +1,11 @@
 DOCKER:= $(shell which docker)
+
+DOCKER_CMD ?= docker run -i --rm --network=$(DOCKER_NETWORK)
+SSH_EXEC ?= $(DOCKER) exec -it
+CONTAINER_SHELL ?= sh
+
+# Connect ssh $host
+define ssh
```

Comment:
this should be `docker_exec`; we can describe it in comments like "it's like ssh for docker"